projects/step-frontend/src/lib/modules/timeseries/chart/time-series-chart.component.scss

```diff
@@ -9,6 +9,23 @@
   //width: 100px !important;
 }
 
+::ng-deep {
+  .u-legend {
+    font-size: 13px;
+    display: flex;
+    justify-content: space-around;
+
+    .u-series {
+      margin-right: 8px;
+      white-space: nowrap;
+
+      .u-marker:empty {
+        display: none;
+      }
+    }
+  }
+}
```

Comment:
if you really can't avoid ::ng-deep use :host to prevent bleed. Better even would be to find a solution to add the styles in the child component

Reply:
the child component is the component genreated by the chart library. I can go with :host instead.

Reply:
:host is not working here..